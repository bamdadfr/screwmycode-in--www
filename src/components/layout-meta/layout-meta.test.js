import React from 'react'
import renderer from 'react-test-renderer'
import LayoutMetaComponent from './layout-meta.component'

describe ('LayoutMetaComponent', () => {

    const Component = <LayoutMetaComponent/>
    const instance = renderer.create (Component).toJSON ()

    it ('should match snapshot', () => {

        expect (instance).toMatchSnapshot ()

    })

})