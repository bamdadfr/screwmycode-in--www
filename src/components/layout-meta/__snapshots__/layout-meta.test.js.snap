// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutMetaComponent should match snapshot 1`] = `null`;
